Update issue templates (#184)
osrf/rmf_demos: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report
about: Create a report to help us improve
title: ''
labels: ''
assignees: ''

---

> 

Reviewer: Add a comment notice about whether or not to ask for technical support via bug reports.

Author: I guess users could use the same form when submitting bug reports or asking for technical support? Unless we want to have separate sections for these?

Reviewer: I recommend being very careful about using GitHub's issues for technical support. We should set up a Discussions thing and point users there, I think.

Author: Thanks for the suggestion. I've added a comment to redirect technical support enquiries to Discussions. 
https://github.com/osrf/rmf_demos/pull/184/commits/ba264e4b71a8c32f03ec6770f7c4baba7d149e1b

**Describe the bug**
A clear and concise description of what the bug is.

> **Workspace (please complete the following information):**
>  - OS:
>  - RMF binaries version: 
>  If building from source,
>  - rmf_demos branch:
>  - rmf_core branch:
>  - rmf_schedule_visualizer branch:
>  - traffic_editor branch:

Reviewer: ```suggestion
**Workspace information:**
 - Operating system and version:
 - ROS distribution:
 - ROS installation method: [from binary packages or from source]
 - RMF installation method: [from binary packages or from source]
 - RMF binaries version, if installed from binaries:
 - RMF source versions, if installed from source:
   - `rmf_demos` branch or commit hash: 
   - `rmf_core` branch or commit hash: 
   - `rmf_schedule_visualizaer` branch or commit hash: 
   - `traffic_editor` branch or commit hash: 
```

> **To Reproduce**

Reviewer: ```suggestion
**Steps to Reproduce**
```

> Steps to reproduce the behavior:

Reviewer: ```suggestion
1. 
2. 
3. ```

**Expected behavior**
A clear and concise description of what you expected to happen.

> 

Reviewer: ```suggestion

**Actual behavior**
A description of what actually happens.

```
